04/app.js

```diff
@@ -1,5 +1,51 @@
 document.addEventListener('DOMContentLoaded', init);
 
 function init() {
-    console.log('DOM');
-}
\ No newline at end of file
+  console.log('DOM');
+
+  const form = document.querySelector('.form');
+  const inputLat = document.querySelector('.form__field--lat');
+  const inputLng = document.querySelector('.form__field--lng');
+
+  const weatherLat = document.querySelector('.weather__lat');
+  const weatherLng = document.querySelector('.weather__lng');
+  const weatherSummary = document.querySelector('.weather__summary');
+  const weatherTemp = document.querySelector('.weather__temperature');
+
+  const apiKey = '022a40fc149e46668af305e2c3de46b0';
```

Comment:
Pamiętaj, aby docelowo nie zamieszczać API KEY na GH, bo ktoś może go przejść. Oczywiście do zadań (i w nie płatnym planie) jest to ok :)